tests/python/integration/test_ewise_fpga.py

```diff
@@ -40,6 +40,8 @@ def check_device(device, host="llvm"):
             b.asnumpy(), np.exp(a.asnumpy()), rtol=1e-5)
 
     check_device("sdaccel")
```

Comment:
In the case which   `XCL_PLATFORM` is not set and `AWS_PLATFORM` is set, dose the above `check_device` do same test of the following `check_device`? It may not occur problem but leave the comment ;)

Reply:
Oh, sorry. On second thought, this may be needed for the test of fallback if not specified device. never mind my comment. Looks good to me.